src/core/public/chrome/ui/header/header.tsx

```diff
@@ -87,7 +87,7 @@ export interface HeaderProps {
   isLocked$: Observable<boolean>;
   loadingCount$: ReturnType<HttpStart['getLoadingCount$']>;
   onIsLockedUpdate: OnIsLockedUpdate;
-  branding: { logoUrl: string };
+  branding: { logoUrl: string | undefined };
```

Comment:
although technically not the same, the same result can be accomplished by `logoUrl?: string` which seems to be common in the repo

Reply:
Good catch! ?: is used a lot in the repo instead of undefined and i think we should follow the same pattern. Will update all to use ?: instead of undefined.